ilastik/widgets/boxListView.py

```diff
@@ -231,35 +227,40 @@ def allowFixIcon(self):
     def allowFixIcon(self, allow):
         self._table.setColumnHidden(self.model.ColumnID.FixIcon, not allow)
 
+    def contextMenuEvent(self, event):
+        menu = QMenu(self)
 
-    def contextMenuEvent(self,event):
-        # idx = self.model.indexAt(event.pos())
-        # box = self.model[idx.row()]
-        menu = QMenu("Menu", self)
-
-        def saveCSVList():
-            import os
-            filename, _filter = QtWidgets.QFileDialog.getSaveFileName(None, 'Save Boxes to txt', os.path.expanduser("~"), ".txt")
-            filename=str(filename)
+        if self.model:
+            export = QAction("Export...", self)
+            export.triggered.connect(self._export_csv)
+            menu.addAction(export)
 
-
-            if filename!="" and self.model!=None:
-                a,b=os.path.splitext(filename)
-                if b!=".txt": filename=a+".txt"
-                self.model.signalSaveAllBoxesToCSV.emit(filename)
-
-
-        export = QAction("Export list of boxes",menu)
-        export.setStatusTip("Export List of boxes")
-        export.triggered.connect(saveCSVList)
-
-        menu.addAction(export)
+        import_ = QAction("Import...", self)
+        import_.triggered.connect(self._import_csv)
+        menu.addAction(import_)
 
         menu.exec_(self.mapToGlobal(event.pos()))
 
+    def _export_csv(self):
+        filename, _filter = QtWidgets.QFileDialog.getSaveFileName(
+            self, "Export Boxes", os.path.expanduser("~"), "CSV (*.csv)", "CSV (*.csv)"
```

Comment:
In GUI mode, we usually default to exporting stuff to the same directory where the project file resides, which is usually the value inside the `WorkingDirectory` slots. Maybe we should be pointing to that directory rather than `~` ?